### DISCUSSION TITLE
V9 Datagrid - prevent row selection

### DISCUSSION THREAD

**Discussion by ivonool** [microsoft/fluentui#31684] [Q&A]
I have a v9 datagrid (multiselection enabled), with a button in a cell. When I click on the button, row will be selected. Any sample how to prevent row selection when clicking on the button? I found a way with Table, but not sure if this is possible with Datagrid.

**Comment by darmageddon** [comment]
Maybe using event.stopPropagation() in button's onClick handler?

onClick={(event) => {
    event.stopPropagation()
}}
